src/app/providers/native-base-provider/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, PropsWithChildren } from 'react';
+import { NativeBaseProvider as NativeBaseProviderBase, extendTheme } from 'native-base';
+
+const NativeBaseProvider: FC<PropsWithChildren> = ({ children }) => {
+  const theme = extendTheme({
+    components: {
+      Input: {
+        variants: {
+          underlined: () => {
+            return {
+              placeholderTextColor: 'light.50',
+              color: '#fff',
+              borderColor: 'light.50',
+              borderBottomWidth: 0.5,
+              _focus: {
+                borderColor: '#fff',
+              },
+            };
+          },
+        },
+      },
+      Text: {
+        baseStyle: {
+          color: '#fff',
+        },
+      },
+      Button: {
+        variants: {
+          solid: () => {
+            return {
+              bg: '#fff',
+              my: 7,
+              w: '50%',
+              _text: {
+                color: 'primary.50',
+                fontWeight: 'bold',
+              },
+            };
+          },
+        },
+      },
+    },
+    colors: {
+      primary: {
+        50: '#0067A0',
+      },
+      light: {
+        50: '#ffffff99',
```

Comment:
<img width="443" alt="image" src="https://user-images.githubusercontent.com/20073193/210327130-e74d729d-7b17-4994-bbca-88b5375144c0.png">

We have a lot of colors in use. Please declare them all here. 
Let's also set "500" as a default or main color in a palette, and if there are different shades of a color we will set a number according to NativeBase guides: https://docs.nativebase.io/default-theme, the darker a color is, the greater the number.